test/api_test.rb

```diff
@@ -304,7 +304,7 @@ class JobWithTags
 
           expected_args_by_version = {
             "5.x" => [1, 2, 3],
-            "6.x" => [{"user" => "gid://app/User/1"}, [1, 2, 3]],
+            "6.x" => [{user: "gid://app/User/1"}, [1, 2, 3]],
```

Comment:
@mperham Now that we support symbols, the API responses can be different. How do you want to handle this?